Mediapipe.Net/Calculators/GpuCalculator.cs

```diff
@@ -34,7 +34,7 @@ public abstract class GpuCalculator<TPacket, T> : Disposable, ICalculator<T>
         private readonly GpuResources gpuResources;
         private readonly GlCalculatorHelper gpuHelper;
         private readonly OutputStreamPoller<GpuBuffer> framePoller;
-        private readonly GCHandle observeStreamHandle;
+        private GCHandle observeStreamHandle;
```

Comment:
...What? Why?

Reply:
gave a warning
![image](https://user-images.githubusercontent.com/46694241/150755978-14cff7ef-10a4-43ea-abd8-6c3d8fe9b515.png)

Reply:
Huh... Didn't know that was a thing